test/JDBC/expected/pivot-vu-verify.out

```diff
@@ -1146,6 +1642,16 @@ int#!#int#!#int#!#int#!#int#!#int
 ~~END~~
 
 
+-- Test view of a stmt with pivot operator
```

Comment:
Please add a line here stating that view with pivot is not yet supported

Reply:
Added comment of "create view with pivot is not yet supported"